src/value/owned.rs

```diff
@@ -583,14 +575,8 @@ impl<'a, T: 'a> Iterator for ElementDataIterator<'a, T> {
     }
 }
 
-impl IonElement for Element {
-    type SymbolToken = Symbol;
-    type Sequence = Sequence;
-    type Struct = Struct;
-    type Builder = Element;
-    type AnnotationsIterator<'a> = SymbolsIterator<'a>;
-
-    fn ion_type(&self) -> IonType {
+impl Element {
```

Comment:
🗺️ For now I've converted the functionality that was associated with the `IonElement` trait into concrete methods on the `Element` type. I expect to refactor these in a follow-on PR.